### DISCUSSION TITLE
WT32-ETH01 Support?

### DISCUSSION THREAD

**Discussion by gellfuen** [UtilitechAS/amsreader-firmware#673] [Q&A] [answered]
Hello!
I don't have WIFI at the place where I want to use AMSReader but I do have Ethernet Lan cable there. Is it possible to install AMSReader on the WT32-ETH01?
<img width="246" alt="WT32-ETH01" src="https://github.com/UtilitechAS/amsreader-firmware/assets/40841094/cd1e370c-8ec3-4ec3-8991-0d20493762cc">
BR

**Comment by gskjold** [comment]
Hi, I can have a look at adding it as a supported board in a future release. Please follow #365 for updates on the progress.

**Reply by gellfuen** [reply]
thx

**Reply by gellfuen** [reply]
I ordered that part now and should get it soon, in the meantime, if you like I attach the datasheet of that WT32-ET01 module:
[WT32-ETH01_datasheet_V1.3+-+en.pdf](https://github.com/UtilitechAS/amsreader-firmware/files/13311586/WT32-ETH01_datasheet_V1.3%2B-%2Ben.pdf)
BR

**Reply by gskjold** [reply, marked as answer]
Thanks, that helps. Added profile in dev firmware, see #365

**Comment by Noschvie** [comment]
@gellfuen maybe you can try to use the WT32-ETH01 device as "wifi range extender" running Tasmota and a separated device running AMS Reader and wifi.

**Reply by gellfuen** [reply]
hmmm, well that idea I still had, but only for workaround... THX at all!
